### PR TITLE
PPPSYS-47023 Fix OpenAPI validation errors

### PR DESCRIPTION
**Pull request by janusz-piechota-piwik**
https://piwikpro.atlassian.net/browse/PPPSYS-47023

### CODE REVIEW

**Review by kororokke — APPROVED** [review]

File: audience_manager/authorized_api/definitions/profile_details.yaml [hunk 1]
```diff
@@ -32,11 +32,11 @@ allOf:
             description: Number of conversions.
             example: 4
           last_revenue:
-            type: float
+            type: number
```
Comment (mrfijas) [inline audience_manager/authorized_api/definitions/profile_details.yaml:35]: can it be
```suggestion
            type: number
            format: float
```
?

**Review by mrfijas — APPROVED** [review]
haven't checked, but I believe that it passes the validation now?

File: audience_manager/authorized_api/definitions/profile_details.yaml (hunk 1 above)
Comment (janusz-piechota-piwik) [reply audience_manager/authorized_api/definitions/profile_details.yaml:35]: If it should be a float then OK. https://swagger.io/docs/specification/data-models/data-types/#numbers

**Comment by janusz-piechota-piwik** [comment]
> haven't checked, but I believe that it passes the validation now?

no errors 
image

File: custom_reports/http_api/definitions/query_request.yaml [hunk 2]
```diff
@@ -178,7 +178,7 @@ properties:
                     - lte
                 value:
                   description: Value that should be matched by filter.
-                  type: float
+                  type: number
```
Comment (mrfijas) [inline custom_reports/http_api/definitions/query_request.yaml:181]: `format: float`?

File: custom_reports/http_api/definitions/query_request.yaml [hunk 3]
```diff
@@ -233,7 +237,7 @@ properties:
       Additional options for query.
     properties:
       sampling:
-        type: float
+        type: number
```
Comment (mrfijas) [inline custom_reports/http_api/definitions/query_request.yaml:240]: float also?

File: custom_reports/http_api/definitions/real_time_query_request.yaml [hunk 4]
```diff
@@ -150,7 +150,7 @@ properties:
                     - lte
                 value:
                   description: Value that should be matched by filter.
-                  type: float
+                  type: number
```
Comment (mrfijas) [inline custom_reports/http_api/definitions/real_time_query_request.yaml:153]: and here?

File: custom_reports/http_api/definitions/real_time_query_request.yaml [hunk 5]
```diff
@@ -205,7 +209,7 @@ properties:
       Additional options for query.
     properties:
       sampling:
-        type: float
+        type: number
```
Comment (mrfijas) [inline custom_reports/http_api/definitions/real_time_query_request.yaml:212]: :hello:

**Review by mrfijas — APPROVED** [review]

**Pull request merged**